web/src/components/MemoEditor/index.tsx

```diff
@@ -75,6 +75,9 @@ const MemoEditor = (props: Props) => {
 
   useEffect(() => {
     editorRef.current?.setContent(contentCache || "");
+    if (user.localSetting.enableEditorAutoFocus) {
+      handleEditorFocus();
+    }
```

Comment:
Let's make it default auto focus.

```suggestion
    handleEditorFocus();
```